Documentation/devel/stage1-implementors-guide.md

```diff
@@ -95,6 +95,12 @@ For example, it removes the network namespace of a pod.
 * `--debug` to activate debugging
 * UUID of the pod
 
+### `rkt stop` => "coreos.com/rkt/stage1/stop"
+
+The stop entrypoint initiates an orderly shutdown of stage1.
+
+In the bundled rkt stage 1, the entrypoint is a statically-linked C program that sends a SIGTERM to the systemd-nspawn process, which will deal with shutting down the container.
+
```

Comment:
Can you say what should stage0 do if the entrypoint is not there in the stage1 manifest? I guess it should just print the error "not implemented" or something.

See #1897.

The manifest below needs to be updated.